alphabert/perception/framebuffer.py

```diff
@@ -1,35 +1,59 @@
 from .frame import Frame
 from .neo4j_reader import Neo4jReader
+from .context_agent import ContextAgent
+
 
 class FrameBuffer:
```

Comment:
For now, the FrameBuffer works with level `1`, only. Happy to refactor it to be able to treat level `2`, as well.